preparing the app for first release

helloseyedjafari opened the pull request.

Commit:
fixed release app crashes

Commit:
fixed release app crashes
Updated Coroutines and MviPresenter.kt Impl
Fixed ViewPagerAdapter Leak

Commit:
fix quranHomeLayoutDirection

Commit:
fix release signing

Commit:
added all translations

Commit:
increased maximum aya/translation font size

Commit:
surahView fontSize now directly work with processor

Commit:
added StartUpView.kt

rezaiyan commented on navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandler.kt:
```diff
@@ -2,13 +2,11 @@ package com.islamversity.navigation.changehandler
+import com.bluelinelabs.conductor.changehandler.androidxtransition.SharedElementTransitionChangeHandler
 import java.util.*
 
 @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```
This class is useless!

helloseyedjafari replied:
removed

rezaiyan commented on navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandlerCompat.kt:
```diff
@@ -1,16 +1,34 @@
-        FadeChangeHandler()
-    )
+    override fun configureSharedElements(container: ViewGroup, from: View?, to: View?, isPush: Boolean) {
+        TODO("Not yet implemented")
```
these TODOs throw `NotImplementedError`

helloseyedjafari replied:
was not calling them
anyways
removed

rezaiyan reviewed (approved):
Thank you

Commit:
removed Controller ChangeHandler
added StartUpView.kt to decide whether we should fill up the DB or it's ready to go

Commit:
added basic OnBoardingView.kt
